presentation/src/main/java/com/yapp/presentation/ui/member/help/Help.kt

```diff
@@ -0,0 +1,226 @@
+        DetailInfoItem(
+            buildAnnotatedString {
+                append(stringResource(id = R.string.help_detail_info_content4_start_text))
+                withStyle(
+                    style = SpanStyle(
+                        fontWeight = FontWeight.Bold,
+                        color = Gray_800
+                    )
+                ) {
+                    append(stringResource(id = R.string.help_detail_info_content4_bold_text))
+                }
+                append(stringResource(id = R.string.help_detail_info_content4_end_text))
+            })
+    }
+}
+
+@Composable
+private fun DetailInfoItem(text: AnnotatedString) {
+    Row {
+        Text(   // 이미지로 교체 예정
+            modifier = Modifier.padding(horizontal = 5.dp),
+            text = "·",
+            fontSize = 12.sp,
+            color = Gray_600
+        )
+        Text(text = text, fontSize = 12.sp, color = Gray_600)
+    }
+}
+
+@Composable
+private fun ScoreRule(type: String, lateScore: Int, absentScore: Int) {
+    Column(modifier = Modifier.padding(top = 28.dp)) {
+        Text(
+            buildAnnotatedString {
+                withStyle(
+                    style = SpanStyle(
+                        fontWeight = FontWeight.Bold,
+                        color = Gray_800,
+                        fontSize = 16.sp
+                    )
+                ) {
+                    append(type)
+                }
+                withStyle(style = SpanStyle(color = Gray_800, fontSize = 16.sp)) {
+                    append(stringResource(id = R.string.help_basic_info_type_text))
+                }
+            }
+        )
+        ScoreRuleTable(tardyScore = lateScore, absentScore = absentScore)
+    }
+}
+
+@Composable
+fun ScoreRuleTable(tardyScore: Int, absentScore: Int) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 12.dp)
+            .clip(RoundedCornerShape(10.dp))
+            .border(BorderStroke(1.dp, Gray_200)),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        ScoreRuleCell(
+            topText = stringResource(R.string.help_basic_info_tardy_text),
+            bottomText = stringResource(R.string.help_basic_info_score_text, tardyScore)
+        )
+
+        ScoreRuleCell(
+            topText = stringResource(R.string.help_basic_info_absent_text),
+            bottomText = stringResource(R.string.help_basic_info_score_text, absentScore)
+        )
+    }
+}
+
+@Composable
+fun RowScope.ScoreRuleCell(
+    topText: String,
+    bottomText: String
+) {
+    Column(
+        modifier = Modifier
+            .weight(1f),
+    ) {
+        Text(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight()
+                .background(Gray_200)
+                .padding(11.5.dp),
```

Comment:
```suggestion
                .padding(12.dp),
```
피그마 스펙이 소숫점으로 끝나는 경우 반올림해서 짝수로 맞춰주심 더 좋을 것 같아요!